client/src/Pages/Watchlater/WatchLater.js

```diff
@@ -1,8 +1,9 @@
-import React,{useEffect} from 'react';
+import React,{useEffect, useState} from 'react';
 import ReactPlayer from 'react-player';
 import { Link,useNavigate } from 'react-router-dom';
 
 import { useVideos } from '../../contexts/VideosContextProvider';
+import Toast from '../../utils/Toast';
 
 const WatchLater = () => {
 
```

Comment:
Try to add a cleanup function in useEffect

Reply:
yeah but here cleanup function is not required